app/Metadata/GitHubFunctions.php

```diff
@@ -69,10 +59,6 @@ private function trim($commit_id)
 	 */
 	public function get_current_branch()
```

Comment:
The PHPdoc comment needs updating. The last sentence

> Return master if it is CI

does not hold anymore.

Reply:
ok, it is updated.